README.md

```diff
@@ -3,13 +3,22 @@
 [![crates.io](https://img.shields.io/crates/v/rustc-hash.svg)](https://crates.io/crates/rustc-hash)
 [![Documentation](https://docs.rs/rustc-hash/badge.svg)](https://docs.rs/rustc-hash)
 
-A speedy, non-cryptographic hashing algorithm used by `rustc` and Firefox.
+A speedy, non-cryptographic hashing algorithm used by `rustc`.
 The [hash map in `std`](https://doc.rust-lang.org/std/collections/struct.HashMap.html) uses SipHash by default, which provides resistance against DOS attacks.
-These attacks aren't as much of a concern in the compiler so we prefer to use the quicker, non-cryptographic Fx algorithm.
-
-The Fx algorithm is a unique one used by Firefox. It is fast because it can hash eight bytes at a time.
-Within `rustc`, it consistently outperforms every other tested algorithm (such as FNV).
-The collision rate is similar or slightly worse than other low-quality hash functions.
+These attacks aren't as much of a concern in the compiler so we prefer to use a
```

Comment:
```suggestion
These attacks aren't a concern in the compiler so we prefer to use a
```
the originally wording bothers me, it's not "as much" of a concern, it is simply not a concern at all